.github/workflows/checkCI.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on: [pull_request]
```

Comment:
If you do not follow a PR based git workflow, you shoul add the push event here, too.